modules/gapi/src/streaming/onevpl/utils.cpp

```diff
@@ -429,4 +432,156 @@ std::string ext_mem_frame_type_to_cstr(int type) {
 } // namespace wip
 } // namespace gapi
 } // namespace cv
+
+// FIXME: find a way to remove this
+#ifdef HAVE_DIRECTX
+#ifdef HAVE_D3D11
+// FIXME: consider handling it in cmake
+#pragma comment(lib,"d3d11.lib")
+
+// get rid of generate macro max/min/etc from DX side
+#define D3D11_NO_HELPERS
+#define NOMINMAX
+#include <d3d11.h>
+#pragma comment(lib, "dxgi")
+#undef NOMINMAX
+#undef D3D11_NO_HELPERS
+
+#endif // HAVE_D3D11
+#endif // HAVE_DIRECTX
+
+namespace {
```

Comment:
Please do not bring up hard platform dependeng links in clear 'utils' component
Move it out please

Reply:
Done